cli/command/formatter/stats.go

```diff
@@ -145,8 +145,10 @@ func (c *containerStatsContext) Container() string {
 
 func (c *containerStatsContext) Name() string {
 	c.AddHeader(nameHeader)
-	name := c.s.Name[1:]
-	return name
+	if len(c.s.Name) > 1 {
+		return c.s.Name[1:]
```

Comment:
I was discussing this with @vdemeester - we should probably have a general function for doing this (out of scope for this PR).

Given that the current implementation is closest to the original code, I think it's ok to keep it as-is (`TrimPrefix` will also be a slightly heavier operation)